release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -84,6 +84,11 @@ For more information, see xref:../installing/installing_aws/installing-aws-china
 
 In {product-title} {product-version}, you can expand an installer provisioned cluster deployed using the `provisioning` network by using Virtual Media on the `baremetal` network. You can use this feature when the `ProvisioningNetwork` configuration setting is set to `Managed`. To use this feature, you must set the `virtualMediaViaExternalNetwork` configuration setting to `true` in the `provisioning` custom resource (CR). You must also edit the machineset to use the API VIP address. See xref:../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.html#preparing-to-deploy-with-virtual-media-on-the-baremetal-network_ipi-install-expanding[Preparing to deploy with Virtual Media on the baremetal network] for details.
 
+id="ocp-4-9-upgrading-etcd-3.4-to-3.5"]
+==== Upgrading etcd from 3.4 to 3.5
+
+OpenShift 4.9 supports etcd 3.5. You must have a valid OpenShift 4.8 backup before upgrading to 4.9. Having a backup ensures a successful upgrade from etcd 3.4 to etcd 3.5.
```

Comment:
nit: having a backup pre-upgrade ensures the admin can restore the cluster in the case of an upgrade failure.

Reply:
Is it necessary to add notes: backup of etcd data is automatic and also points out the backup data location, such as:
/etc/kubernetes/cluster-backup/upgrade-backup-2021-09-01_050044

Reply:
+1 we can also identify that state is part of the etcd-operator status.